opensrp-reveal/src/test/java/org/smartregister/reveal/server/FileHTTPServerTest.java

```diff
@@ -0,0 +1,57 @@
+package org.smartregister.reveal.server;
+
+import android.content.Context;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.robolectric.RuntimeEnvironment;
+import org.smartregister.reveal.BaseUnitTest;
+import org.smartregister.reveal.R;
+import org.smartregister.reveal.util.OfflineMapHelper;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Richard Kareko on 2/6/20.
+ */
+
+public class FileHTTPServerTest extends BaseUnitTest {
+    Context context = RuntimeEnvironment.application;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4217760)